zegami_sdk/collection.py

```diff
@@ -259,6 +289,23 @@ def download_single(index, url):
         for i in range(len(images)):
             ordered.append(images[i])
         return ordered
+    
+    def get_tags(self):
```

Comment:
Could this be hidden as `_get_tags`? Thinking from an outsider, they wouldn't know what to do with tag indices.

Reply:
Or better still `_get_tag_indices`?